backend/src/services/v2/registry.ts

```diff
@@ -52,6 +56,19 @@ export async function listImageTags(user: UserDoc, imageRef: RepoRef) {
 }
 
 export async function listModelImages(user: UserDoc, modelId: string) {
```

Comment:
I was going to recommend adding a test for this but I can't seem to find any registry service tests...?